Makefile

```diff
@@ -13,14 +13,14 @@ include Makefile.global
 all: extension
 
 # build extension
-extension:
+extension: $(citus_abs_srcdir)/src/include/citus_version.h
 	$(MAKE) -C src/backend/distributed/ all
 install-extension: extension
 	$(MAKE) -C src/backend/distributed/ install
 install-headers: extension
 	$(MKDIR_P) '$(DESTDIR)$(includedir_server)/distributed/'
 # generated headers are located in the build directory
-	$(INSTALL_DATA) src/include/citus_config.h '$(DESTDIR)$(includedir_server)/'
+	$(INSTALL_DATA) $(citus_abs_srcdir)/src/include/citus_config.h '$(DESTDIR)$(includedir_server)/'
```

Comment:
Can you make this install `citus_version.h`, instead? I don't think we need `citus_config.h` anywhere, and `autoconf` discourages installing the generated config file (probably because it contains bare `PACKAGE_NAME`, etc. variables that will conflict if everyone installs their `config.h` file).

We don't include it and no one else in GitHub does. It's probably a mistake it was ever included here.